content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -72,8 +72,63 @@ In case your cluster nodes are customized and install additional security and
 telemetry agents on the node, make sure to check with the vendor of the agent whether it has dependency on Docker.
 
 ### Telemetry and security agent vendors
+This section is intended to collect information about various telemetry and security agents that may have dependency on containers runtime. Support matrix section outlines the supported runtimes and Migration from dockershim section is designed to help users transition from dockershim to other container runtimes.
```

Comment:
nit: Add a new line after the heading and before the first sentence.
nit: The security agent vendors listed in this section are third-party vendors? If so, add the third-party shortcode.

I'm not entirely clear on the first two sentences.

Reply:
```
[Migration from dockershim](#migration-from-dockershim) section ...
```